Core/GDCore/IDE/Dialogs/LayoutEditorCanvas/LayoutEditorCanvasOptions.cpp

```diff
@@ -35,20 +38,26 @@ void LayoutEditorCanvasOptions::SerializeTo(SerializerElement& element) const {
   element.SetAttribute("gridB", gridB);
   element.SetAttribute("zoomFactor", zoomFactor);
   element.SetAttribute("windowMask", windowMask);
+  element.SetAttribute("viewXPosition", viewXPosition);
+  element.SetAttribute("viewYPosition", viewYPosition);
+  element.SetAttribute("viewPositionValid", viewPositionValid);
 }
 
 void LayoutEditorCanvasOptions::UnserializeFrom(
     const SerializerElement& element) {
   grid = element.GetBoolAttribute("grid");
   snap = element.GetBoolAttribute("snap");
   windowMask = element.GetBoolAttribute("windowMask");
+  viewPositionValid = element.GetBoolAttribute("viewPositionValid", false);
   gridWidth = element.GetIntAttribute("gridWidth", 32);
   gridHeight = element.GetIntAttribute("gridHeight", 32);
   gridOffsetX = element.GetIntAttribute("gridOffsetX", 0);
   gridOffsetY = element.GetIntAttribute("gridOffsetY", 0);
   gridR = element.GetIntAttribute("gridR", 158);
   gridG = element.GetIntAttribute("gridG", 180);
   gridB = element.GetIntAttribute("gridB", 255);
+  viewXPosition = element.GetIntAttribute("viewXPosition", 400);

```

Comment:
Same, let's put 0 as default value, because 400 and 300 are giving false hints that these numbers are important.